ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/task/InitProjectTask.groovy

```diff
@@ -30,13 +30,13 @@ class InitProjectTask extends HubTask {
         println("############################################\n")
         println(" - Set username and password")
         println("     There are several ways to do this. The easiest is to set mlUsername and mlPassword in gradle.properties.")
-        println("     For other approaches see: https://github.com/marklogic/marklogic-data-hub/wiki/Password-Management\n")
+        println("     For other approaches see: https://docs.marklogic.com/datahub/security/set-security-credentials-using-gradle.html\n")
         println(" - To deploy your application into MarkLogic...")
         println("     gradle mlDeploy\t# this will bootstrap your application")
         println("     gradle mlLoadModules\t# this will load your custom plugins into MarkLogic\n")
         println(" - Full list of gradle tasks:")
-        println("     https://marklogic.github.io/marklogic-data-hub/docs/gradle-tasks\n")
+        println("     http://docs.marklogic.com/tools/gradle/gradle-tasks.html\n")
         println(" - Curious about the project structure?")
```

Comment:
http://docs.marklogic.com/tools/gradle/gradle-tasks.html is currently broken and there is no mention about that in DHFPROD-2286.

Reply:
oops, typo'd.

Should no longer be broken.